src/dbnode/storage/repair/options.go

```diff
@@ -29,54 +29,61 @@ import (
-	errInvalidRepairTimeJitter      = errors.New("invalid repair time jitter in repair options")
-	errTimeOffsetOrJitterTooBig     = errors.New("repair time offset plus jitter should be no more than repair interval")
-	errInvalidRepairCheckInterval   = errors.New("invalid repair check interval in repair options")
-	errRepairCheckIntervalTooBig    = errors.New("repair check interval too big in repair options")
-	errInvalidRepairThrottle        = errors.New("invalid repair throttle in repair options")
-	errInvalidRepairMaxRetries      = errors.New("invalid repair max retries in repair options")
-	errNoHostBlockMetadataSlicePool = errors.New("no host block metadata pool in repair options")
+	errNoAdminClient                           = errors.New("no admin client in repair options")
+	errInvalidRepairInterval                   = errors.New("invalid repair interval in repair options")
+	errInvalidRepairTimeOffset                 = errors.New("invalid repair time offset in repair options")
+	errInvalidRepairTimeJitter                 = errors.New("invalid repair time jitter in repair options")
+	errTimeOffsetOrJitterTooBig                = errors.New("repair time offset plus jitter should be no more than repair interval")
+	errInvalidRepairCheckInterval              = errors.New("invalid repair check interval in repair options")
+	errRepairCheckIntervalTooBig               = errors.New("repair check interval too big in repair options")
+	errInvalidRepairThrottle                   = errors.New("invalid repair throttle in repair options")
+	errInvalidRepairMaxRetries                 = errors.New("invalid repair max retries in repair options")
+	errNoHostBlockMetadataSlicePool            = errors.New("no host block metadata pool in repair options")
+	errInvalidDebugShadowComparisonsPercentage = errors.New("debug shadow comparisons percentage must be between 0 and 1")
 )
 
 type options struct {
-	adminClient                client.AdminClient
-	repairConsistencyLevel     topology.ReadConsistencyLevel
-	repairShardConcurrency     int
-	repairInterval             time.Duration
-	repairTimeOffset           time.Duration
-	repairTimeJitter           time.Duration
-	repairCheckInterval        time.Duration
-	repairThrottle             time.Duration
-	repairMaxRetries           int
-	hostBlockMetadataSlicePool HostBlockMetadataSlicePool
+	adminClient                      client.AdminClient
+	repairConsistencyLevel           topology.ReadConsistencyLevel
+	repairShardConcurrency           int
+	repairInterval                   time.Duration
+	repairTimeOffset                 time.Duration
+	repairTimeJitter                 time.Duration
+	repairCheckInterval              time.Duration
+	repairThrottle                   time.Duration
+	repairMaxRetries                 int
+	hostBlockMetadataSlicePool       HostBlockMetadataSlicePool
+	debugShadowComparisonsEnabled    bool
+	debugShadowComparisonsPercentage float64
```

Comment:
`float32` should be fine?

Reply:
doesn't hurt to have a more accurate representation